docs/rackhd/static_file_server.rst

```diff
@@ -0,0 +1,94 @@
+.. _static-file-server-label:
+
+Static File Service Setup
+---------------------------
+.. _on-http: https://github.com/RackHD/on-http
+.. _nginx: https://www.nginx.com/
+.. _install nginx: https://www.nginx.com/resources/wiki/start/topics/tutorials/install/
+.. _nginx_conf: https://www.nginx.com/resources/wiki/start/topics/examples/full/
+.. _config.json: https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/monorail/files/config.json
+.. _httpProxies: http://rackhd.readthedocs.io/en/latest/rackhd/configuration.html?highlight=httpProxies
+.. _httpStaticRoot: http://rackhd.readthedocs.io/en/latest/rackhd/configuration.html?highlight=httpStaticRoot
+
+
+There are two kinds of static files in RackHD, and this section introduces a mechanism to move one type of
+them (files for discovery and OS installation) to a separate third-party service in order to offload the burden
+of file transmission in RackHD.
+
+Files That can be Moved into a Separate Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Some files, including schema, swagger configuration and others, interacts closely with RackHD, and are part of
+its functionalities. Others are served for node discovery and OS installation (if users put OS image under the
+same static file directory).  `on-http`_ manages all the files mentioned above by default, and the latter
+(**files for discovery and OS installation**) can be moved to a third-party static file server, which will be
+discussed below.
+
+Diagrams for Different Working Modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+RackHD supports three modes to serve static files.
+
+Legacy Mode: nodes get static files from `on-http`_ service (default).
+Single-Host Mode: nodes get static files from another service in the same host as RackHD.
```

Comment:
Although the reStruct file looks like a new line for 'Single-Host Mode' but the rendered html shows it's in the same line as 'Legacy Mode'. Suggest to give a new line, maybe even add bullets :)